apps/stata/stata.talon

```diff
@@ -0,0 +1,7 @@
+# Commands for the Stata Do-File Editor
+os: windows
+app: stata
```

Comment:
this needs an and before app, otherwise it is treated they are ored and would be active on all windows systems

Reply:
Are you sure about this? From https://talon.wiki/Customization/talon-files: "Requirements of different types (e.g. 'app' and 'os') are AND-ed together."

 Using the debug window, the stata commands do not seem to be active when I'm not focusing a stata window.

Reply:
Ya sorry you are right, confused myself last night because I happened to be looking at https://github.com/talonhub/community/blob/main/core/application_matches.py around the same time and it uses `and` everywhere, which I guess is all redundant lol.